exam1/exam1/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace exam1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int sum;
+            string str;
+            while (true)
+            {
+                Console.WriteLine("Введите сумму: ");
+                str = Console.ReadLine();
+                if (!int.TryParse(str, out sum))
+                {
+                    Console.WriteLine("Ошибка ввода, вы ввели не число, повторите поыптку!");
+                }
+                else
+                {
+                    if (int.Parse(str) <= 0 || int.Parse(str) >= 50000 || int.Parse(str) % 100 != 0)

```

Comment:
Неверное неравенство. Сумма 50 000 руб. допустима.
```suggestion
                    if (int.Parse(str) <= 0 || int.Parse(str) > 50000 || int.Parse(str) % 100 != 0)

```